From 3cb5a971cc3cda7eac4c1f75bdda164ba61f2a20 Mon Sep 17 00:00:00 2001
From: Matthias Schmitz <matthias.schmitz@port4949.net>
Date: Tue, 17 Dec 2024 23:53:55 +0000
Subject: [PATCH] imx93_evk/spl.c: Set dram_timing_1866mts unconditional

Signed-off-by: Matthias Schmitz <matthias.schmitz@port4949.net>
---
 board/freescale/imx93_evk/spl.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/board/freescale/imx93_evk/spl.c b/board/freescale/imx93_evk/spl.c
index 5ee850e2850..5ef9584c768 100644
--- a/board/freescale/imx93_evk/spl.c
+++ b/board/freescale/imx93_evk/spl.c
@@ -69,8 +69,7 @@ void spl_dram_init(void)
 {
 	struct dram_timing_info *ptiming = &dram_timing;
 #if IS_ENABLED(CONFIG_IMX93_EVK_LPDDR4X)
-	if (is_voltage_mode(VOLT_LOW_DRIVE))
-		ptiming = &dram_timing_1866mts;
+	ptiming = &dram_timing_1866mts;
 #endif
 
 	printf("DDR: %uMTS\n", ptiming->fsp_msg[0].drate);
-- 
2.39.5

